src/include/distributed/multi_logical_optimizer.h

```diff
@@ -38,6 +38,7 @@
 #define HLL_ADD_AGGREGATE_NAME "hll_add_agg"
 #define HLL_UNION_AGGREGATE_NAME "hll_union_agg"
 #define HLL_CARDINALITY_FUNC_NAME "hll_cardinality"
+#define HLL_ENABLE_HASH_GUC_NAME "hll.enable_hashagg"
```

Comment:
I think the name enable_hashagg can be confused with PostgreSQL's enable_hashagg, We wouldn't want to explain all users the difference between the enable_hashagg and hll.enable_hashagg. Can we name it something else; force_groupagg maybe?